### DISCUSSION TITLE
internet access on eth0

### DISCUSSION THREAD

**Discussion by itzika** [RaspAP/raspap-webgui#1026] [Q&A]
Hi, What im trying to do is to connect the Raspap wirelessly to the main modem/router, and to connect a desktop pc to the raspap via eth cable (so i can have internet on the desktop pc).

What i did thus far is 
step 1: setup raspap with all defaults...connected wirelessly to the main router and have internet access on wlan0. 
step 2: connected my pc to the raspap via eth cable. no internet.

Im really not sure which mode should i have in order for this setup to work.
can someone please guide me to the right place/settings etc?

thanks.

**Comment by billz** [comment]
What you're describing is bridging the wireless interface in client mode, which is not supported by the 802.11 standard.
Attempts to do so will throw an "operation not permitted" error.

Long discussion and much hair-pulling here https://forums.raspberrypi.com/viewtopic.php?f=36&t=132674

You're welcome to try it, but RaspAP does not/will not support this.
